source/main/gui/panels/GUI_TopMenubar.cpp

```diff
@@ -716,8 +716,19 @@ void RoR::GUI::TopMenubar::DrawActorListSinglePlayer()
     {
         Actor* player_actor = App::GetSimController()->GetPlayerActor();
         int i = 0;
+        int x = 0;
         for (auto actor : actor_list)
         {
+            char text_buf_rem[200];
+            snprintf(text_buf_rem, 200, "X" "##[%d]", x++);
```

Comment:
You can use `i` here, just don't include the `++` post-increment operator 😄 

The whole thing works like this:
```
int i = 0;
for (...) { 
    snprintf(i); 
    snprintf(i);
    i++;
}
```